src/zenml/integrations/airflow/orchestrators/airflow_orchestrator.py

```diff
@@ -44,7 +44,9 @@
 from zenml.orchestrators.utils import get_orchestrator_run_name
 from zenml.stack import StackValidator
 from zenml.utils import daemon, io_utils
-from zenml.utils.pipeline_docker_image_builder import PipelineDockerImageBuilder
+from zenml.utils.pipeline_docker_image_builder import (
+    PipelineDockerImageBuilder,
+)
```

Comment:
This might indicate that something in the ruff config is off, the old line was exactly 80 chars so it should not have been reformatted.

Reply:
Ah I think this changed the max length to 79 now, which actually makes sense since that is PEP8 standard anyways

Reply:
Everything is 79 now.